modules/core/arrow-free/src/main/kotlin/arrow/free/FreeApplicative.kt

```diff
@@ -62,7 +65,8 @@ inline fun <reified F, A> FreeApplicativeOf<F, A>.foldK(FA: Applicative<F> = app
                 override fun <A> invoke(fa: Kind<F, A>): Const<M, A> = f(fa).fix()
             }, Const.applicative(MM)).value()
 
-    fun monad(): Free<F, A> = foldMap(Free.functionKF(), Free.applicativeF()).fix()
+    // FIXME(paco) Free.applicativeF is broken
```

Comment:
how is it broken?

Reply:
Global lookup of arbitrary instance. A no-no in this new model. I'm still unsure how to fix it.

`private val applicative: Applicative<FreePartialOf<F>> = arrow.typeclasses.applicative()`

Reply:
I did the simple fix of requiring it as a parameter. We'll figure the rest out later.